app/src/app/components/stix/stix-list/stix-list.component.ts

```diff
@@ -437,9 +450,18 @@ export class StixListComponent implements OnInit, AfterViewInit, OnDestroy {
             let target_ref = element['target_ref'];
             // Get type to navigate from target_ref
             let type = this.typeMap[target_ref.split('--')[0]];
-
             this.router.navigateByUrl('/'+ type + '/' + target_ref);
         }
+        else if (this.config.clickBehavior && this.config.clickBehavior == "linkToObjectRef") {
+          // technically a note can be linked to many objects, we will select the first object
+          let object_ref = element['object_refs'][0];
+          // Get type to navigate from target_ref
+          let type = this.typeMap[object_ref.split('--')[0]];
+
+          this.sidebarService.opened = true;
+          this.sidebarService.currentTab = 'notes';
+          this.router.navigateByUrl('/'+ type + '/' + object_ref);
```

Comment:
This is a great addition! I like that it opens the sidebar directly to the `notes` tab on the object page. This makes it super easy to view the context.

Reply:
Thanks :)